doc/api/process.md

```diff
@@ -452,7 +452,7 @@ added: v0.1.27
 
 The `process.argv` property returns a array containing the command line
 arguments passed when the Node.js process was launched. The first element will
-be 'node', the second element will be the name of the JavaScript file. The
+be  [`process.execPath()`], the second element will be the name of the JavaScript file. The
```

Comment:
While we're in here, we may as well change "name of the JavaScript file" to something like "path to the JavaScript file."